api/src/opentrons/protocol_engine/execution/pipetting.py

```diff
@@ -243,6 +243,37 @@ async def dispense_in_place(
 
         return volume
 
+    async def aspirate_in_place(
+        self,
+        pipette_id: str,
+        volume: float,
+        flow_rate: float,
+    ) -> float:
+        """Aspirate liquid without moving the pipette."""
+        hw_pipette = self._state_store.pipettes.get_hardware_pipette(
+            pipette_id=pipette_id,
+            attached_pipettes=self._hardware_api.attached_instruments,
+        )
+
+        ready_to_aspirate = self._state_store.pipettes.get_is_ready_to_aspirate(
+            pipette_id=pipette_id,
+            pipette_config=hw_pipette.config,
+        )
+
+        if not ready_to_aspirate:
+            raise RuntimeError(
+                "When aspirate is called on something other than a "
+                "well relative position, we can't move to the top of"
+                " the well to prepare for aspiration. This might "
+                "cause over aspiration if the previous command is a "
+                "blow_out."
```

Comment:
Let's simplify this message:

```suggestion
                "Pipette cannot aspirate in place because of a previous blow out."
                " The first aspirate following a blow-out must be from a specific well"
                " so the plunger can be reset in a known safe position."
               
```